src/vit-servicing-station/vit-servicing-station-tests/src/common/clients/rest/raw.rs

```diff
@@ -50,6 +50,14 @@ impl RestClient {
         .map_err(Error::RequestError)
     }
 
+    pub fn put_raw_snapshot(&self, tag: &str, content: String) -> Result<Response, Error> {
+        self.put(
+            &self.path_builder.clone().admin().raw_snapshot(tag),
+            content,
+        )
+        .map_err(Error::RequestError)
```

Comment:
NITPICK: shouldn't it be easy to write it like: `.map_err(Into::into)`

Reply:
done